scripts/generate-authors.sh

```diff
@@ -52,6 +52,7 @@ cd "$(dirname "$(readlink -f "$BASH_SOURCE")")/.."
 	Samin Muhammad Ridwanul Karim
 	Stefan Robert
 	Bernhard Tempel
+	Samuel Robertson
```

Comment:
No need to add yourself. You'll be added correctly by `generate-authors.sh` based on the data in your commit, In this place in `generate-authors.sh`, the authors contributing in our [subversion](https://subversion.apache.org/) times on sourceforge are tracked. With git, we don't need that system any more.